nmma/em/analysis.py

```diff
@@ -632,9 +638,11 @@ def analysis(args):
         trigger_time=trigger_time,
         tmin=args.tmin,
         tmax=args.tmax,
+        dt=args.dt,
```

Comment:
what is this dt used for? does it has to be the same as the sample_times' dt?

Reply:
if not, it would be best with another time interval

Reply:
sorry, this shouldn't be there. I forgot to remove this from likelihood and analysis